serialization.md

```diff
@@ -1,7 +1,7 @@
 # OCI Image Serialization
 
 An *Image* is an ordered collection of root filesystem changes and the corresponding execution parameters for use within a container runtime.
-This specification outlines the format of these filesystem changes and corresponding parameters and describes how to create and use them for use with a container runtime and execution tool.
+This specification outlines the JSON format describing images for use with a container runtime and execution tool and its relationship to filesystem change sets, described in [Layers](layer.md).
```

Comment:
This paragraph seems like it's either covered in the terminology section below or [in the README][1].  Can we drop the paragraph, [move the terminology to a glossary][2], and get straight into the serialization schema/example?  If that's too much to bite off in this PR, I'm happy to file it in a follow-up PR.

[1]: https://github.com/opencontainers/image-spec/blame/v0.4.0/README.md#L5
[2]: https://github.com/opencontainers/image-spec/pull/206/files#r76134009